vcd/resource_vcd_vapp_vm_multi_test.go

```diff
@@ -1,5 +1,4 @@
-//go:build multivm || ALL || functional
-// +build multivm ALL functional
+//go:build multivm || functional
```

Comment:
So, isn't `ALL` required in this case?

Reply:
Not required, as the main test will work all the same. We had a recent case of ALL vs functional and we determined that ALL doesn't always do what it means :)
But ALL will be needed anyway. Adding it.